connector_cli/tripletexlogin.py

```diff
@@ -32,6 +35,12 @@ def login_via_tripletex(sesam_node, args):
             secrets = {
                 "sessionToken": data["value"]["token"],
             }
+
+            if manifest.get("requires_service_api_access"):
+                secrets["service_api_access"] = args.service_jwt
```

Comment:
Same comment about the name of the secret.

Reply:
done